src/ucp/core/ucp_ep.c

```diff
@@ -874,27 +874,19 @@ static void ucp_ep_close_flushed_callback(ucp_request_t *req)
                                       &req->send.disconnect.prog_id);
 }
 
-static int ucp_ep_close_is_param_flag(const ucp_request_param_t *param,
-                                      int flag)
-{
-    return (param->op_attr_mask & UCP_EP_PARAM_FIELD_FLAGS) &&
-           (param->flags == flag);
-}
-
 static ucs_status_t
-ucp_ep_close_nbx_check_params(ucp_ep_h ep, const ucp_request_param_t *param)
+ucp_ep_close_nbx_check_params(ucp_ep_h ep, int force)
```

Comment:
int *force
since this function checks the params it makes more sense it would return force rather that accept it

Reply:
unhandled?